tests/common/tlspolicy/tlspolicy_controller_test.go

```diff
@@ -116,7 +122,7 @@ var _ = Describe("TLSPolicy controller", func() {
 
 			By("creating a valid Gateway")
 			gateway = tests.NewGatewayBuilder("test-gateway", gatewayClass.Name, testNamespace).
-				WithHTTPListener("test-listener", "test.example.com").Gateway
+				WithHTTPListener("test-listener", newHostName()).Gateway
```

Comment:
Was this needed for running them in parallel?

Reply:
No, originally there seemed to be a test flake with the changes for some reason so I was seeing will this change make it more stable. But it look stable without these changes now, so I'll revert these changes

Reply:
Reverted